spec/acceptance/place_order_spec.rb

```diff
@@ -114,81 +116,84 @@
         billing_email_address: 'barry@gmail.com'
       }
 
-      items = [
-        { id: '456', name: 'Drones', price: '144', quantity: '34' },
-        { id: '454', name: 'Uranium', price: '130', quantity: '100' },
-        { id: '767', name: 'ACHSDJVHJDWVVFWVYEUVFW', price: '10', quantity: '200000' },
-        { id: '999', name: 'Screws', price: '0.9', quantity: '2000' }
-      ]
+      item1 = { id: '456', name: 'Drones', price: '144', quantity: '34' }
+      item2 = { id: '454', name: 'Uranium', price: '130', quantity: '100' }
+      item3 = { id: '767', name: 'ACHSDJVHJDWVVFWVYEUVFW', price: '10', quantity: '200000' }
+      item4 = { id: '999', name: 'Screws', price: '0.9', quantity: '2000' }
 
       save_customer_details.execute(customer_details: customer_details)
-      save_items_details.execute(items: items)
+      save_items_details.execute(item: item1)
```

Comment:
Would it be possible to call `save_items_details.execute` once but with multiple arguments?

Reply:
I think we want to call it multiple times because that is what will happen in production when a user adds multiple item rows